File: Zoos.cpp
#include <iostream>
#include <string.h>
using namespace std;
int main() {
	string name;
	
	cin >> name;
	int z = 0, o = 0;

	for(char i : name){
		if (i == 'z'){
			z = z + 1;
		}
		else if(i == 'o'){
			o = o + 1;
		}
	}
	z*2 == o? cout<<"Yes"<<endl:
	cout<<"No"<<endl;		


	return 0;
}

File: Zoos.py
obj = input()

def chkword(name):
    z = 0
    o = 0
    for i in name:
        if i == 'z':
            z+=1
        elif i == 'o':
            o+=1
    print("Yes" if z*2 == o else "No")

if len(obj) <= 20:
    chkword(obj)
